tests/test_bs.py

```diff
@@ -0,0 +1,229 @@
+        time.sleep(1)
+        total_sleep_time += 1
+
+
+def create_deployment(file_name,name_space, **kwargs):
+    with open(path.join(path.dirname(__file__), file_name)) as f:
+        dep = yaml.safe_load(f)
+        if kwargs:
+            dep = update_bootsrap_node_in_client(dep,
+                                                 kwargs['bootstrap_ip'],
+                                                 kwargs['bootstrap_port'],
+                                                 kwargs['bootstrap_key'])
+
+        k8s_beta = client.ExtensionsV1beta1Api()
+        resp1 = k8s_beta.create_namespaced_deployment(body=dep, namespace=name_space)
+        wait_to_deployment_to_be_ready(resp1.metadata._name, name_space)
+
+        return resp1
+
+
+def delete_deployment(deployment_name, name_space):
+    k8s_beta = client.ExtensionsV1beta1Api()
+    resp = k8s_beta.delete_namespaced_deployment(name=deployment_name,
+                                                 namespace=name_space,
+                                                 body=client.V1DeleteOptions(
+                                                     propagation_policy='Foreground',
+                                                     grace_period_seconds=5))
+    return resp
+
+
+def query_bootstrap_es(indx, namespace, bootstrap_po_name):
+    es = Elasticsearch(['http://127.0.0.1:9200'])
+    fltr = Q("match_phrase", kubernetes__namespace_name=namespace) & \
+           Q("match_phrase", kubernetes__pod_name=bootstrap_po_name) & \
+           Q("match_phrase", M="Local node identity")
+    s = Search(index=indx, using=es).query('bool', filter=[fltr])
+    hits=list(s.scan())
+    for h in hits:
+        match = re.search(r"Local node identity \w+ (?P<bootstarap_key>\w+)", h.log)
+        if match:
+            return match.group('bootstarap_key')
+    return None
+
+
+def query_es_client_bootstrap(indx, namespace, client_po_name):
+    es = Elasticsearch(['http://127.0.0.1:9200'])
+    fltr = Q("match_phrase", kubernetes__namespace_name=namespace) & \
+           Q("match_phrase", kubernetes__pod_name=client_po_name) & Q("match_phrase", M='discovery_bootstrap')
+    s = Search(index=indx, using=es).query('bool', filter=[fltr])
+    hits = list(s.scan())
+
+    print("Number of Hits: {0}".format(len(hits)))
+    s=set([h.N for h in hits])
+    return (len(s))
+
+
+def query_es_gossip_message(indx, namespace, client_po_name):
+    es = Elasticsearch(['http://127.0.0.1:9200'])
+    fltr = Q("match_phrase", kubernetes__namespace_name=namespace) & \
+           Q("match_phrase", kubernetes__pod_name=client_po_name) & Q("match_phrase", M='new_gossip_message')
+    s = Search(index=indx, using=es).query('bool', filter=[fltr])
+    hits = list(s.scan())
+
+    print("Number of gossip Hits: {0}".format(len(hits)))
+    s=set([h.N for h in hits])
+    return (len(s))
+
+
+@pytest.fixture(scope='session')
+def load_config():
+    config.load_kube_config('/Users/isaac/.kube/config')
```

Comment:
this path won't work for everyone, maybe add it as param or require the file to be in the execution folder